eth/vm/code_stream.py

```diff
@@ -46,31 +46,31 @@ def __getitem__(self, i: int) -> int:
 
     def __iter__(self) -> Iterator[int]:
         # a very performance-sensitive method
-        pc = self.pc
+        pc = self.program_counter
```

Comment:
I would extract this to it's own PR.  :+1:

Reply:
Ooops..too late. I though the approval and rage merged